OpenSim/Simulation/Test/testFrames.cpp

```diff
@@ -332,6 +341,45 @@ void testPhysicalOffsetFrameOnBodySerialize()
         "testPhysicalOffsetFrameOnBodySerialize(): incorrect MobilizedBodyIndex");
 }
 
+void testPhysicalOffsetFrameOnPhysicalOffsetFrameOrder()
+{
+    Model pendulum("double_pendulum.osim");
+
+    SimTK::Transform X_RO;
+    X_RO.setP(SimTK::Vec3(0.1, 0.2, 0.3));
+    X_RO.updR().setRotationFromAngleAboutAxis(SimTK::Pi / 4.0, SimTK::ZAxis);
```

Comment:
No significance whatsoever. I *borrrowed* the code block from the `testPhysicalOffsetFrameOnBodySerialize()` case earlier in `testFrames`. The outcome of the transform is not being tested, so it doesn't matter what value it has..